beetsplug/parentwork.py

```diff
@@ -173,13 +173,17 @@ def find_work(self, item, force):
             return
 
         hasparent = hasattr(item, 'parentwork')
-        if force or not hasparent:
+        workfalse = True
```

Comment:
What does this variable represent? The `workfalse` name doesn't evoke a specific condition.

Reply:
```workfalse``` says if ```mb_workid``` and ```mb_workid_current``` have the same value, i. e. if the parentwork corresponds to the work or if the work has been changed since the parentwork was retrieved.

Reply:
Sounds good—but can you maybe pick a name that reflects that use? Maybe something like work_changed?